Add FAX.PLUS

alohi opened the pull request.

alohi committed:
Add FAX.PLUS

kmpoppe committed:
Renamed image to faxplus.png after soze reduction

Carlgo11 commented on _data/other.yml:
```diff
@@ -212,6 +212,13 @@ websites:
+    url: https://www.fax.plus
+    img: faxplus.png
+    tfa:
+      - software
```
```suggestion
      - totp
```

kmpoppe replied:
Resolved with 94e0792. Why didn't I see that 😐

kmpoppe committed:
software -> totp

Change as per https://github.com/2factorauth/twofactorauth/pull/4120#discussion_r336756836
